Convert_MyGene_to_OMIM.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Gene symbol to MyGene ID function, and MyGene ID to OMIM ID converter function
+
+# Modules required:
+import requests
+
+
+
+# Function to retrieve MyGene ID from Gene common symbol:
+def get_MyGene(gene):
+    '''This function takes a gene symbol as an input, and returns the MyGene ID number for the gene.
+    If there is not a match in the MyGene database, an error message is printed that the symbol is not valid.'''
+    r = requests.get(f"https://mygene.info/v3/query?q={gene}&fields=all&size=10&from=0&fetch_all=false&facet_size=10&entrezonly=false&ensemblonly=false&dotfield=false")
+    r = r.json()
+    try: 
+        MyGene_ID = r["hits"][0]["_id"]
+        return MyGene_ID
+    except:
+        print("Error: Invalid MyGene ID")
```

Comment:
Look into logger really useful vs print! I'll let you figure out why  :)

Reply:
Sounds good! I'll make some changes